File: plugin/src/api/selection.ts
export namespace ZO {
	export class SelectionFile {
		version: number;
		selection: any;

		constructor(version: number) {
			this.version = version;
			this.selection = undefined;
		}
	}

	export class Selection {
		library_id: number;
		document_id: number;

		constructor(library_id: number, document_id: number) {
			this.library_id = library_id;
			this.document_id = document_id;
		}
	}
}

File: importer/src/api/select/output/selection.rs
use serde::Serialize;

#[derive(Debug, Clone, Serialize)]
pub struct SelectionOutput {
	pub version: i64,
	pub selection: Selection
}

#[derive(Debug, Clone, Serialize)]
pub struct Selection {
	pub library_id: i64,
	pub document_id: i64
}